wine_cellar/apps/wine/templates/wine_card.html

```diff
@@ -2,13 +2,15 @@
 <div class="wine_card__content">
     <a href="{% url 'wine-detail' wine.pk %}">
         <div class="pure-g">
-            <div class="pure-u-8-24">
-                <img src="{{ wine.get_type_image }}"
-                     height="85%"
+            <div class="pure-u-9-24">
+                <img src="{{ wine.image }}"
```

Comment:
- I think we need more padding to the right, otherwise it's too close to the text
- If the image is not full width it's left aligned, I think centred within the container would make more sense